schema/dmt/parse_test.go

```diff
@@ -8,53 +8,47 @@ import (
 	"github.com/ipld/go-ipld-prime/node/bindnode"
 	"github.com/ipld/go-ipld-prime/schema"
 	schemadmt "github.com/ipld/go-ipld-prime/schema/dmt"
-	// "github.com/ipld/go-ipld-prime/schema/schema2"
 )
 
-// TestSchemaSchemaParse takes the schema-schema.json document -- the self-describing schema --
-// and attempts to unmarshal it into our code-generated schema DMT types.
-//
-// This is *not* exactly the schema-schema that's upstream in the specs repo -- yet.
-// We've made some alterations to make it conform to learnings had during implementing this.
-// Some of these alterations may make it back it up to the schema-schema in the specs repo
-// (after, of course, sustaining further discussion).
-//
-// The changes that might be worth upstreaming are:
-//
```

Comment:
Well okay, aside all else, I gotta just say I love seeing a bunch of todos and wishlist items going poof into a block of done -> gone.

Reply:
:dancers: